apps/codeforafrica/src/components/SinglePartner/SinglePartner.js

```diff
@@ -0,0 +1,89 @@
+import { Section } from "@commons-ui/core";
```

Comment:
Why a new component? What where we using before?

Reply:
@kilemensi , before this component was implemented in a [page](https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/src/pages/about/%5Bunit%5D/index.page.js)

Reply:
Not sure I follow @koechkevin. If this was build in a page, it means it's just a collection of other components, no?

Reply:
@koechkevin ?

Reply:
Yes, it is a collection of other components but there's some extra styles added to the components so I thought a standalone page would work best @kilemensi